packages/react-ui-components/src/DropDown/style.css

```diff
@@ -59,6 +59,7 @@
 }
 .dropDown__contents--isOpen {
     display: block;
+    outline: 6px solid var(--colors-PrimaryBlue);
```

Comment:
Maybe box-shadow (see bellow) can be more "correct" here (accessibility), but not sure. Visually both works.